src/Controllers.php

```diff
@@ -33,6 +33,13 @@ public function connect(Silex\Application $app)
             ->value('page', '')
             ->assert('page', '.*');
 
+        $ctr->get('/sitemap/{version}.xml', [$this, 'sitemap'])
+            ->bind('sitemap')
+            ->value('version', '');
```

Comment:
Can I remove this one too, then?

https://github.com/bolt/docs/blob/site/src/Controllers.php#L32